packages/empty-state/src/FeaturesEmptyState/FeaturesEmptyState.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+export const rootStyles = css`
+  max-width: 648px;
```

Comment:
Same as above - it's just the value that looks good. It doesn't even necessarily need to be set since each feature box inside has a `max-width` of `200px`, and it doesn't need an explicit variable since it's only used once in the component.